ledfx/integrations/mood_manager.py

```
@@ -0,0 +1,1104 @@
+        if self._structure_analyzer is not None:
+            if section != self._last_section or event is not None:
+                if section is not None and section != self._last_section:
+                    self._last_section = section
+                    _LOGGER.info(f"Section changed: {section.value}")
+
+                # Fire structure change event
+                try:
+                    self._ledfx.events.fire_event(
+                        StructureChangeEvent(section, event)
+                    )
+                except Exception as e:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)